G2P.pm

```diff
@@ -633,8 +634,8 @@ sub variants_filtered_by_frequency_threshold {
   my $variants = shift;
   my @pass_variants = ();
   foreach my $variant (@$variants) {
-    if (!defined $self->{highest_frequencies}->{$variant} ||
-         $self->{highest_frequencies}->{$variant} <= $af_threshold ||
+    my $highest_frequency = $self->highest_frequency($variant);
```

Comment:
Here it's fetching the highest frequency, I would add a comment saying that. Makes it easier to understand when it's get or set.